example/sonivoxrender.c

```diff
@@ -93,6 +94,9 @@ int initializeLibrary(void)
 	setmode(fileno(stdout), O_BINARY);
 #endif
 
+    EAS_SetDebugFile(stderr, 1);
+    EAS_SetDebugLevel(_EAS_SEVERITY_WARNING);
```

Comment:
When building in release mode, CMake adds -DNDEBUG to the build flags. I suggest to use it:

```suggestion
#ifdef NDEBUG
    EAS_SetDebugLevel(_EAS_SEVERITY_NOFILTER);
#else
    EAS_SetDebugLevel(_EAS_SEVERITY_WARNING);
#endif
```